docs/rfc/active/data-permissions/README.md

```diff
@@ -0,0 +1,128 @@
+- Start Date: 2022-04-19
+- RFC PR:
+- Discussion Issue: 
+- Implementation PR(s): 
+
+# Modeling Data Permissions
+
+## Summary
+
+Many datahub sources provide access control mechanisms to manage user access to assets.
+Datahub should provide functionality for viewing these permissions. 
+
+## Motivation
+
+Within many sql systems, the data control language provides commands like GRANT, DENY and REVOKE to manage control to datasets.
+
+Being able to view grants is the first step toward supporting larger integrations with dataset permissions. [See Here](https://feature-requests.datahubproject.io/b/feedback/p/access-control-on-datasets-bi-assets)
+
+Of the datahub integration sources listed, the following have some form of data access control management:
+ - Athena (via IAM)
+ - BigQuery
+ - Clickhouse
+ - Druid
+ - Elastic Search
+ - Glue
+ - Hive (via Apache Ranger)
+ - Kafka
+ - Looker
+ - MariaDB
+ - Metabase
+ - MongoDB
+ - MicrosoftSQLServer
+ - MySQL
+ - Oracle
+ - PostgreSQL
+ - Redash
+ - Redshift
+ - S3 (via IAM)
+ - Sagemaker
+ - Snowflake
+ - Superset
+ - Tableau
+ - Trino
+
+## Requirements
+
+The permissions aspect must be:
+- Appliable to datasets, containers, dashboards
+- Support ingesting and viewing structured access control (GRANT {permission} ON {object} TO {user or role}
+- Support ingesting and viewing unstructured access control (IAM Policy Documents, Custom Policies from other datasets)
+- 
+### Extensibility
+
+> Please also call out extensibility requirements. Is this proposal meant to be extended in the future? Are you adding
+> a new API or set of models that others can build on in later? Please list these concerns here as well.
+
+## Non-Requirements
+
+This RFC specifically calls for functionality to view the permissions. Actions taken on the permissions may be great areas for future enhancement but is not a focus here.
+
+## Detailed design
+
+There are no new Entities associated with this design.
```

Comment:
When modeling data quality rules (assertions), we went through a similar debate, started with an aspect-oriented design, but later moved towards modeling assertions as top-level entities with ability to attach to one or more datasets etc. I think it may be a good idea to do the same with Permissions, since they can attach to a single dataset or groups of logical datasets, etc.